pkg/sysched/README.md

```diff
@@ -0,0 +1,256 @@
+  image: nginx:1.16
+```
+NOTE: In order for the binding webhook to work, one must need to label the namespace (for the seccomp profile and 
+binding CRDs) using `kubectl label` command as follows. The `default` namespace is used here.
+
+``kubectl label ns default spo.x-k8s.io/enable-binding=``
+
+Following commands create the binding CRDs for `nginx`, `memcached`, and `redis` with their corresponding seccomp
+profile CRDs.
+
+```
+kubectl apply -f manifests/sysched/examples/nginx.binding.yaml
+kubectl apply -f manifests/sysched/examples/memcached.binding.yaml
+kubectl apply -f manifests/sysched/examples/redis.binding.yaml
+```
+
+To see the created profile binding CRDs, please issue the following command: `kubectl get profilebindings`.
+
+```
+NAME                AGE
+memcached-binding   56m
+nginx-binding       56m
+redis-binding       56m
+```
+
+#### Step 3: Creating deployments
+The third step is to create deployments. The following yaml file (`nginx1.yaml`) shows the deployment
+yaml for `nginx` where the scheduler the `default-scheduler` with `SySched` plugin enabled. If the
+`SySched` plugin is enabled and used through a secondary scheduler please specify the `schedulerName`
+field in the specification.
+
+```
+$ cat manifests/sysched/examples/nginx1.yaml
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx1
+  labels:
+    app: nginx1
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nginx1
+  template:
+    metadata:
+      labels:
+        app: nginx1
+    spec:
+      containers:
+      - name: nginx1 
+        image: nginx:1.16
+        ports:
+        - containerPort: 80
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: nginx1
+spec:
+  type: NodePort
+  selector:
+    app: nginx1
+  ports:
+    - protocol: TCP
+      port: 80
+```
+
+The following commands create the deployments for two instance of `nginx`, one instance of `memcached`, and one
+instance of `redis`. Please note that the order of the pod deployment may result in different placement of pods.
+
+```
+kubectl apply -f manifests/sysched/examples/nginx1.yaml
+kubectl apply -f manifests/sysched/examples/redis1.yaml
+kubectl apply -f manifests/sysched/examples/nginx2.yaml
+kubectl apply -f manifests/sysched/examples/memcached1.yaml
+```
+
+To see the placement result, please issue the following command: `kubectl get pods -o wide`
+
+```
+NAME                          READY   STATUS    RESTARTS   AGE   IP            NODE       NOMINATED NODE   READINESS GATES
+memcached1-5fd5d4cf7f-j5gvr   1/1     Running   0          4s    10.244.1.86   mynode-1   <none>           <none>
+nginx1-b86d6f76c-bg4hl        1/1     Running   0          40s   10.244.2.93   mynode-2   <none>           <none>
+nginx2-d678b7967-9jrb5        1/1     Running   0          18s   10.244.2.94   mynode-2   <none>           <none>
+redis1-54fcc8f949-5ss5k       1/1     Running   0          32s   10.244.1.85   mynode-1   <none>           <none>
+```
```

Comment:
I guess you may want to add a short stanza explaining why it's scheduled this way?